2021-08-26 19:57:52,338 INFO    MainThread:14268 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 19:57:52,338 INFO    MainThread:14268 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 19:57:52,338 INFO    MainThread:14268 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 19:57:52,338 INFO    MainThread:14268 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210826_195752-2r2u8tfk\logs\debug.log
2021-08-26 19:57:52,339 INFO    MainThread:14268 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210826_195752-2r2u8tfk\logs\debug-internal.log
2021-08-26 19:57:52,339 INFO    MainThread:14268 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002CE1A376280>
2021-08-26 19:57:52,366 INFO    MainThread:14268 [wandb_init.py:init():375] calling init triggers
2021-08-26 19:57:52,366 INFO    MainThread:14268 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-26 19:57:52,366 INFO    MainThread:14268 [wandb_init.py:init():424] starting backend
2021-08-26 19:57:52,367 INFO    MainThread:14268 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-08-26 19:57:52,372 INFO    MainThread:14268 [backend.py:ensure_launched():135] starting backend process...
2021-08-26 19:57:52,502 INFO    MainThread:14268 [backend.py:ensure_launched():139] started backend process with pid: 17412
2021-08-26 19:57:52,503 INFO    MainThread:14268 [wandb_init.py:init():429] backend started and connected
2021-08-26 19:57:52,504 INFO    MainThread:14268 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-26 19:57:52,504 INFO    MainThread:14268 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-26 19:57:52,504 INFO    MainThread:14268 [wandb_init.py:init():477] updated telemetry
2021-08-26 19:57:52,505 INFO    MainThread:14268 [wandb_init.py:init():500] communicating current version
2021-08-26 19:57:56,402 INFO    MainThread:14268 [wandb_init.py:init():505] got version response 
2021-08-26 19:57:56,402 INFO    MainThread:14268 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-26 19:57:57,230 INFO    MainThread:14268 [wandb_init.py:init():540] starting run threads in backend
2021-08-26 19:58:02,246 INFO    MainThread:14268 [wandb_run.py:_console_start():1584] atexit reg
2021-08-26 19:58:02,247 INFO    MainThread:14268 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-26 19:58:02,248 INFO    MainThread:14268 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-26 19:58:02,250 INFO    MainThread:14268 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-26 19:58:02,251 INFO    MainThread:14268 [wandb_init.py:init():565] run started, returning control to user process
2021-08-26 19:58:13,049 INFO    MainThread:14268 [wandb_init.py:_pause_backend():256] pausing backend
2021-08-26 19:58:13,050 INFO    MainThread:14268 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 19:59:24,777 INFO    MainThread:14268 [wandb_init.py:_resume_backend():265] resuming backend
2021-08-26 19:59:24,788 INFO    MainThread:14268 [wandb_init.py:_pause_backend():256] pausing backend
2021-08-26 19:59:24,788 INFO    MainThread:14268 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 19:59:31,035 INFO    MainThread:14268 [wandb_init.py:_resume_backend():265] resuming backend
2021-08-26 19:59:31,077 INFO    MainThread:14268 [wandb_init.py:_pause_backend():256] pausing backend
2021-08-26 19:59:31,077 INFO    MainThread:14268 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 20:00:05,685 INFO    MainThread:14268 [wandb_init.py:_resume_backend():265] resuming backend
2021-08-26 20:00:05,883 INFO    MainThread:14268 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 20:00:05,886 INFO    MainThread:14268 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 20:00:05,888 INFO    MainThread:14268 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 20:00:05,891 INFO    MainThread:14268 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210826_200005-r8coolt8\logs\debug.log
2021-08-26 20:00:05,891 INFO    MainThread:14268 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210826_200005-r8coolt8\logs\debug-internal.log
2021-08-26 20:00:05,894 INFO    MainThread:14268 [wandb_init.py:init():375] calling init triggers
2021-08-26 20:00:05,898 INFO    MainThread:14268 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-26 20:00:05,900 INFO    MainThread:14268 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 2r2u8tfk
2021-08-26 20:00:05,900 INFO    MainThread:14268 [wandb_run.py:finish():1254] finishing run khushal/pipweather-recognization/2r2u8tfk
